packages/bundler-plugin-core/src/utils/normalizeOptions.ts

```diff
@@ -84,8 +106,8 @@ const optionsSchemaFactory = (options: Options) =>
       .number({
         invalid_type_error: "`retryCount` must be a number.",
       })
-      .positive({
```

Comment:
Sorry I was wrong about this!! I re-read `fetchWithRetry` and the first time we hit the url is (re)try #1 so how you had it before was correct